### DISCUSSION TITLE
ISQCMC '25 [Palermo, IT] (28-29 October, 2025)

### DISCUSSION THREAD

**Discussion by omarcostahamido** [Quantumland-art/community#81] [Events]
**Where:** Palermo, Palazzo Steri, University of Palermo, Italy
**When:** 28-29 October, 2025
**URL:** https://2025.isqcmc.org/
<img width="1013" alt="Screenshot 2024-11-16 at 11 06 11 AM" src="https://github.com/user-attachments/assets/bf72ac88-2589-4749-a5a1-dba285f12b52">

Conference chairs:	
- Maria Mannone (CNR, Italy & University of Potsdam, Germany)
- Lucia Rizzuto (University of Palermo, Italy)

Advisory Board:	
- Eduardo Reck Miranda (University of Plymouth & Moth)
- Karl Jansen (DESY, Germany)
- Maciej Lewenstein (ICFO - Institute of Photonic Sciences, Spain)

Organizational Team:	
- Maria Mannone (CNR, Italy & University of Potsdam, Germany)
- Omar Costa Hamido (CEIS20, University of Coimbra, Portugal)
- Paulo Itaborai (DESY, Germany)

Cultural Programme Curator:	
- Harry Kumar (Moth, UK)

[International peer-review committee](https://2025.isqcmc.org/peer-review-committee.html)

Registration: free

Call for papers

- Deadline for paper submission:	~March 1, 2025~ Sunday, May 18 2025 - 23:59 CEST
- Deadline for music submission: Sunday, June 15 2025 - 23:59 CEST
- Page limit:	5 pages, [ISQCMC 2025 template](https://www.overleaf.com/latex/templates/isqcmc-2025-template/qyfngrjtyvjv)
- Notice of acceptance:	~June 15, 2025~ August 15, 2025
- Deadline for the registration:	~July 14, 2025~ September 12, 2025
- Submission: [email](mailto:quantummusicsubmission@gmail.com?subject=%5BISQCMC%20%2725%5D%20Paper%20Submission) the PDF of your contribution to quantummusicsubmission AT gmail DOT com

University of Palermo, Department of Physics and Chemistry “Emilio Segrè”, in partnership with Moth

**Comment by omarcostahamido** [comment]
Paper submission is over but the music submission is open!

> _Deadline for music submission: | Sunday, June 15 2025 - 23:59 CEST_

See more details at https://2025.isqcmc.org/submissions.html#music

**Comment by omarcostahamido** [comment]
🚨 ISQCMC '25: registrations are open now!
Registration is FREE but seats are _very_ limited.
[2025.isqcmc.org](http://2025.isqcmc.org/)

**Comment by omarcostahamido** [comment]
🚨 ISQCMC '25: program is now online!
[2025.isqcmc.org/program](https://2025.isqcmc.org/program)

**Comment by omarcostahamido** [comment]
Some reports, news, links, and other shares about the event (feel free to add yours below):
- https://www.unipa.it/III-International-Symposium-on-Quantum-Computing-and-Musical-Creativity/
- https://x.com/omarcostahamido/status/1983224136731160966
- https://bsky.app/profile/och.art/post/3m4btfvneks2h
- https://www.mathisintheair.org/wp/2026/01/quantum-musica-creativita-e-pace/

**Comment by medusamedusa** [comment]
Proceedings online! The articles accepted for ISQCMC ’25 are now available through Zenodo, and they can be accessed online:
Scientific Contributions —> https://isqcmc.org/Proceedings
Musical Contributions —> https://isqcmc.org/Music

**Reply by omarcostahamido** [reply]
🎉 this deserves a post of its own in the [📚 bookshelf section](https://github.com/orgs/Quantumland-art/discussions/categories/bookshelf)

**Comment by medusamedusa** [comment]
I also would like to share a recently-published blog post, where ISQCMC ’25 is also mentioned: https://www.mathisintheair.org/wp/2026/01/quantum-musica-creativita-e-pace/
The post is in Italian, but that’s an easy read, with plenty of pictures.
